h/static/styles/partials-v2/_search.scss

```diff
@@ -66,10 +66,21 @@
 
 .search-result-zero {
   background: $grey-2;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  padding: 10px;
+  padding: 30px;
+}
+
+.search-result-zero__title {
+  margin-top: 0;
+}
+
+.search-result-zero__list {
+  list-style-position: inside;
+  padding-left: 0;
+}
+
+.search-result-zero a {
+  color: inherit;
+  text-decoration: underline;
```

Comment:
A couple of classes used on links in the new website stuff have these two CSS rules now, it seems to be how most links in Conor's designs look, maybe should move onto the default link style for the new website stuff?

Also, I've used `text-decoration: underline;` to be consistent with e.g. links in the search page sidebar, but `border-bottom: 1px solid black;` actually looks more similar to the design I think (bigger distance between bottom of text and "underline")